python/opentrons_ot3_firmware/messages/payloads.py

```diff
@@ -154,6 +154,16 @@ class ReadPresenceSensingVoltageResponsePayload(utils.BinarySerializable):
     gripper: utils.UInt16Field
 
 
+@dataclass
+class ToolsDetectedNotificationPayload(utils.BinarySerializable):
+    """A tool detection notification for tools attached to z axis, a axis and gripper."""
+
+    # Tools are mapped to an enum
+    z_motor: utils.UInt8Field
```

Comment:
These are `uint8` and the c++ side is `uint16` i believe. we should also ideally just make these `enum` elements with custom ser/des.